src/util/misc.rs

```diff
@@ -59,14 +59,15 @@ pub fn script_find_and_remove(haystack: &mut Vec<u8>, needle: &[u8]) -> usize {
 
 /// Hash message for signature using Bitcoin's message signing format
 pub fn signed_msg_hash(msg: &str) -> sha256d::Hash {
-    sha256d::Hash::hash(
-        &[
-            MSG_SIGN_PREFIX,
-            &encode::serialize(&encode::VarInt(msg.len() as u64)),
-            msg.as_bytes(),
-        ]
-        .concat(),
-    )
+    let msg_len = encode::VarInt(msg.len() as u64);
```

Comment:
Again, is there a reason for this to be here instead of right before where it's inputted?

Reply:
Done